openfl/federated/task/runner_keras.py

```diff
@@ -17,8 +18,8 @@
 
 with catch_warnings():
     simplefilter(action="ignore")
+    import keras as ke
     import tensorflow as tf
```

Comment:
we should get rid of this import. I think the only place where `tf` is called is in `reset_opt_vars()`

Reply:
done